docs/source/user_guide/automl.ipynb

```diff
@@ -435,6 +435,28 @@
     "automl.results"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parallel AutoML Search\n",
+    "\n",
+    "The `AutoMLSearch` class is also capable of performing the search in parallel using an underlying Dask engine.  To utilize this feature, simply specify the \"parallel\" engine argument. The desired number of workers can also be specified using the \"engine_workers\" argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# using this pipeline parameter means that our Imputer components in the pipelines will only search through 'median' and 'most_frequent' stretegies for 'numeric_impute_strategy'\n",
+    "automl = AutoMLSearch(X_train=X, y_train=y, problem_type='binary', pipeline_parameters=pipeline_hyperparameters, \n",
+    "                      engine=\"parallel\", engine_workers = 4)\n",
+    "automl.search()\n",
+    "automl.best_pipeline.hyperparameters"
```

Comment:
When I've dealt with parallelism before, you always have to ask the question : "is the functional kernel I'm parallelizing worth the overhead in worker spinup/management?"  It seems to me in this specific case, it isn't.  And nope, I haven't done perf tests on this in specific.  I can generate some but I might need some help/suggestions in figuring out how best to make the pipeline train/score take longer as that's the function that's being parallelized.  If I can come up with a good example where perhaps the train/score function takes on the order of 1-2 minutes, we might be able to see the search process benefit from Dasking.  Any suggestions on maybe a large dataset I can search() my way through?

Reply:
OK, so see below for some rudimentary tests I ran locally on the fraud dataset.   Set up a basic for loop to scan through datasets with 10, 100, 1000, 2000, 4000 and 8000 rows.  Below are the timing results of the full search on those data subsets.

I saved and looked at the scores of the best pipelines and they were all the same.

![image](https://user-images.githubusercontent.com/64713315/110390029-5d2f4b00-8033-11eb-9b5b-cfb4d0125424.png)

Reply:
I think this was with the default settings

Reply:
Results look good to me! I like how it's clear here that running more workers results in a faster fit time, and especially since the scores are the same in the two runs, the engine seems to work as we would expect!